source/guide/index.md

```diff
@@ -6,99 +6,156 @@ order: 1.1
 version: 2.1
 ---
 
+<!-- toc -->
+
 ## What is Weex?
 
-> Weex is A framework for building Mobile cross-platform UIs.
+> **Weex is a framework for building high-performance mobile apps with modern web development experience.**
+
+Weex dedicates to enable developers to use modern web development experience to build both Android, iOS and web applications with a single codebase. In practice, you can use javascript and modern front-end frameworks to develop mobile apps after integrating the WeexSDK.
 
-Weex enables developers to use web development experience to build both Android, iOS and web apps with a single codebase.
+The structure of Weex is decoupled, the render engines are separate from the syntax layer. Weex doesn't rely on any specific front-end frameworks, but mainly Weex supports [Vue.js](https://vuejs.org/) and [Rax](https://alibaba.github.io/rax/).
+
+Another primary goal of Weex is to keep up with modern development technologies and platform capabilities both on web and native. Productivity and performance can coexist in Weex. Writing Weex pages feels the same like writing web pages. Rendering Weex pages is just same with rendering native pages.
 
 ## Overview
 
-Here is an example writing in Weex and Vue.js:
+If you just want to try Weex, You don't need to install anything. There is an [online playground](http://dotwe.org/vue/) of Weex, you can write single page examples on it without any configurations. The source code should be writing in Vue.js [single file component](https://vuejs.org/v2/guide/single-file-components.html) syntax, and the render result of the web platform will be displayed in the mock phone shell.
+
+Here is an [example](http://dotwe.org/vue/8da01827631b21150a12dd54d7114380) written in Weex and Vue.js:
 
 ![Weex Example](./images/weex-example-yo.png)
 
-This example is written in Vue.js [single file component](https://vuejs.org/v2/guide/single-file-components.html) syntax. Vue.js 2.0
+This example rendered a latter "Yo" in the center of the screen. If you want to preview the render result on the real mobile device, you could install the [Weex playground app](https://weex-project.io/playground.html) or integrate Weex SDK into your own app. And then, scan the QR code on the right of the online playground you will see the result rendered on your own device.
```

Comment:
This example renders...
on the mobile device, you need to install the ...
Then scan your page's QR code with your playground app or your own app (with the QR scan feature to load scanned URL with Weex SDK), and you will see the rendered result on your device.